### ISSUE TITLE
Ultra96 v2 bdf

### ISSUE THREAD

**Issue by DavideConficconi** [Avnet/bdf#3]
Hello,

I noticed that the speed grade among Ultra96v2 BDF  revision 1.1 changed from "e" to "i" w.r.t. revision 1.0.
Is this correct? 

Thank you in advance

Regards

**Comment by narrucmot** [comment]
Yes, that is correct.  With the recent release of the industrial version of the Ultra96-V2, and because we also needed to make other edits and fixes to this BDF file, we decided to also update the target Zynq UltraScale+ 3EG device to the ‘-i’ temp rating.  More information about these changes can be found at the following links.
https://www.element14.com/community/groups/fpga-group/blog/2020/08/13/free-vitis-20201-training-for-ultra96-v2

https://ohwr.org/project/soc-course/wikis/Custom-Board-Definition-Files#fixing-the-ultra96-v2-board-files

--Tom
